app/src/main/java/ar/com/wolox/android/example/ui/example/ExampleFragment.kt

```diff
@@ -2,13 +2,20 @@ package ar.com.wolox.android.example.ui.example
 
 import ar.com.wolox.android.R
 import ar.com.wolox.android.databinding.FragmentExampleBinding
+import ar.com.wolox.android.example.utils.Extras
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
 
 class ExampleFragment private constructor() : WolmoFragment<FragmentExampleBinding, ExamplePresenter>(), ExampleView {
 
+    companion object {
+        fun newInstance() = ExampleFragment()
```

Comment:
If you plan to reuse this class, you should change its name to something more accurate